src/Core/Employee/Access/EmployeeFormAccessCheckerInterface.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * 2007-2019 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
```

Comment:
@matks I'm thinking to run the license fixing command `prestashop:licenses:update` but scared that it will update licenses that are out of scope of my PR, wdyt?

Reply:
What about using a shell script to 1) extract all modified files by your PR in a whitelist 2) run `prestashop:licenses:update` 3) perform `git checkout` on all files NOT in your whitelist

So it will result in `prestashop:licenses:update` being applied only to your PR.